api/src/components/link/controller.ts

```diff
@@ -74,9 +74,11 @@ export const deleteLink = async (event: I.AuthenticatedAPIRequest<undefined, und
 
         if (
             link.publicationFromRef.currentStatus !== 'DRAFT' ||
-            !link.publicationFromRef.publicationStatus.every((status) => status.status === 'DRAFT')
+            !link.publicationFromRef.publicationStatus.every((status) => status.status !== 'LIVE')
```

Comment:
is this check really needed? since no publication can go back from LIVE to DRAFT.

currentStatus !== 'DRAFT' should be enough

Reply:
Might be needed, might not be, always been in, since there is mention of this 'HIDDEN' status type, in which LIVE publications can go from LIVE -> HIDDEN -> LIVE, not sure if that is still planned or not